### DISCUSSION TITLE
Get button from the community models page doesn't actually get anything but the model name

### DISCUSSION THREAD

**Discussion by sprior** [open-webui/open-webui#7472] [Issues]
I've noticed that in Open-WebUI if I go to the models screen and select Discover a model from the OpenWebUI Community, then view a model there and click the Get button, I don't actually get a model imported that has any of the metadata specified in the community page as I'd expect.  To do what I expected the Get button to do I have to download it as a JSON export, save it to my local machine, then use the import function in the GUI.  Is this a bug or am I missing something?

**Comment by jordanlambrecht** [comment]
I'm also experiencing this.

**Comment by drphungky** [comment]
Also experiencing this on a self hosted instance with a domain address, e.g. http://ai.domain.com does not work, however entering my local IP address and port does work. When entering with the domain, I get an example script instead.

**Reply by drphungky** [reply]
;Actually, just figured it out: using https://ai.domain.case DOES work. Using http however redirects to https, and then I get the example script. Very odd behavior, but reproducible.
